lol-admin/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM rust:latest
+
+# needed for lol-core
+RUN rustup component add rustfmt
+
+# given the existing project structure, from lol-admin dir
+# you run from one dir up to capture lol-core dependency
+# docker build -t lol-admin -f Dockerfile ..
+
+# then we have access to whole project for lol-core dependency
+COPY ./ /lol
+# TODO: could parameterize this so it could target lol-monitor
+# though here we only want to build admin
+WORKDIR /lol/lol-admin
```

Comment:
We should be able to `cargo build` project without moving to the sub directory.

Reply:
Ah I thought that would build everything including the other bins we don't want.

Reply:
Oh. You can build the specific binary by `cargo build --bin lol-admin`.

You are doing `cargo build` and this may result in build every binaries I guess.